src/main/java/seedu/contax/logic/commands/EditPriorityCommand.java

```diff
@@ -14,17 +14,21 @@
 
 
 /**
- * Edits the level of an existing priority in the appointment list with level low, medium or high.
+ * Edits the priority level of an existing priority in the appointment list with level low, medium or high.
  */
 public class EditPriorityCommand extends Command {
     public static final String COMMAND_WORD = "priority";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the level of priority.\n"
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Edits the level of priority of an existing appointment in the appointment list .\n"
```

Comment:
Please see how markdown is done in the other commands, and follow the syntax there.

Reply:
thx edited